### DISCUSSION TITLE
TP5 - Producto Escalar - Pregunta sobre el prototipo de la función

### DISCUSSION THREAD

**Discussion by AlenCrayt** [INGCOM-UNRN-P1/cursada-2023#179] [Preguntas y respuestas] [answered]
En el punto 2 del TP5 tiene que ser si o si la función 
```C
int producto_escalar(int filas, int columnas, int matriz[filas][columnas], int escalar)
```
o puede ser algo diferente como un `void`?

**Comment by martinvilu** [comment, marked as answer]
Y, siendo que `matriz`puede ser `NULL` y las dimensiones `0` o negativas. Usa el retorno para indicar posibles fallos.
